swagger2/src/main/java/com/cerner/beadledom/swagger2/SwaggerApiResource.java

```diff
@@ -0,0 +1,42 @@
+package com.cerner.beadledom.swagger2;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.jaxrs.listing.AcceptHeaderApiListingResource;
+import io.swagger.models.Swagger;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletContext;
+import javax.ws.rs.GET;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Application;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+
+/**
+ * This provides the swagger 2 documentation at the /api-docs path that was previously used for
+ * swagger 1.
+ */
+@Path("/api-docs")
+public class SwaggerApiResource extends AcceptHeaderApiListingResource {
```

Comment:
extend from BaseApiListingResource since you aren't using anything from AcceptHeaderApiListingResource

Reply:
https://github.com/cerner/beadledom/pull/119/commits/6cc9839dbbd326009806d5ebe61c225f6a3a295b#diff-5761bd845b64ea0dbf59487084ba0800L24